code/modules/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -0,0 +1,345 @@
+    Robert has reported a further increase in frequency of the strange, prying visitors who ask questions they have no buisness asking. I've requested him to keep everything on strict lockdown and have permanetly dismissed all other assistants.<br>
+    I've also instructed him to use the encryption method we discussed for any important quantitative data. The poor lad... I don't think he truly understands what he's gotten himself into...<br>
+    It's clear what happens now. One day they'll show up uninvited, and claim my research as their own, leaving me as nothing more than a bullet ridden corpse floating in space.<br>
+    I can't stick around to the let that happen.<br>
+    I'm escaping into the very thing that brought all this trouble to my doorstep in the first place - my hotel.<br>
+    I'll be in <u>FGeo</u> (That will make sense to anyone who should know)<br>
+    I'm sorry that I must go like this. Maybe one day things will be different and it will be safe to return... maybe...<br>
+    Goodbye<br>
+	<br>
+	<i>Doctor Hilbert</i>"}
+
+/obj/item/paper/crumpled/robertsworkjournal
+    name = "Work Journal"
+    info = {"<h4>First Week!</h4>
+	First week on the new job. It's a secretarial position, but hey, whatever pays the bills. Plus it seems like some intersting stuff goes on here.<br>
+	Doc says its best that I don't openly talk about his research with others, I guess he doesn't want it getting out or something. I've caught myself slipping a few times when talking to others, it's hard not to brag about someting this cool!<br>
+	I'm not really sure why I'm choosing to journal this. Doc seems to log everything. He says it's incase he discovers anything important.<br>
+    I guess thats why I'm doing it too, I've always wanted to be a part of something important.<br>
+    Here's to a new job and to becoming a part of something important!<br>
+	<h4>Weird times...</h4>
+	Things are starting to get a little strange around here. Just weeks after Doc's amazing breakthrough, weird visitors have began showing up unannouced, asking strange things about Doc's work.<br>
+    I knew Doc wasn't a big fan of company, but even he seemed strangely unnerved when I told him about the visitors.<br>
+    He said it's important that from here on out we keep tight security on everthing, even other staff members.<br>
+    He also said something about securing data, something about sixty-four bases. What's that mean? Something to do with baseball? Doc never struct me as the sports type...<br>
+    He often uses a lot of big sciencey words that I don't really understand, but I kinda dig it, it makes me feel like I'm witnessing something big.<br>
+    I hope things go back to normal soon, but I guess that's the price you pay for being a part of something important.<br>
+	<h4>Last day I guess?</h4>
+	Thinks are officially starting to get too strange for me.<br>
```

Comment:
Should probably be "Things" instead of "Thinks"

Reply:
yes thank you